cypress/integration/forms/RadioControl/can_set_a_value.feature

```diff
@@ -1,7 +1,7 @@
-Feature: The RadioGroup can set a value
+Feature: The RadioControl can set a value
 
     Scenario: The user clicks the first option
-        Given a required RadioGroup with no selected value
-        And the RadioGroup has two options
+        Given a required RadioControl with no selected value
+        And the RadioControl has two options
```

Comment:
These steps seem inaccurate in my opinion. The Given refers to a story that actually renders a GroupControl with three required RadioControls.

The And mentions two options, but there are actually three options. Plus the statement in the And is not linked to the story or asserted, but it actually refers to the syntax declared in the story. That seems to be the cause for these statements getting out of sync with what's actually being tested.

I've talked about this with @Mohammer5 before, and he and I differ in opinion on this matter, but I'd personally prefer to condense the Given and And to a Single Given:

```
Given a GroupControl with three required RadioControls with no selected value
```

Is it uglier? Yes.

But now the step clearly lists the assumptions that this step makes. So in the step definition you know what statements you have to assert to ensure that the feature doesn't get out of sync with the actual test.

In addition I'd then change the story title to reflect exactly what the story should contain: `('Testing:RadioControl', 'Default')` is too generic. Just from the story title you don't know what it's supposed to contain.

That way the assumptions are clearly conveyed from feature to steps to story, and there is less chance of the three getting out of sync, like what happened here.

Reply:
> The Given refers to a story that actually renders a GroupControl with three required RadioControls.

OOPS!

Reply:
We also tackle the gist of the above later (that is, if everyone agrees), and just fix the minimal amount to not stall this PR too much, if you prefer.

Reply:
Yeah, I think that would be good to do.

Reply:
If you want I could help, I'm a bit blocked with the scheduler until ui@5 releases anyway. Let me know if I can do anything.

Reply:
@ismay thanks, that would be great !

Reply:
I'll create a PR based on this PR, with some minimal fixes. Afterwards we can create an issue for the rest.

Reply:
This is just for the radio tests: https://github.com/dhis2/ui/pull/47. It's just a quick fix.

Reply:
Created a follow up issue for my above comment here: https://github.com/dhis2/ui/issues/48